petshop/styles.css

```diff
@@ -1,3 +1,147 @@
 body {
-    background-color: aqua;
-}
\ No newline at end of file
+    font-family: Arial, sans-serif;
+    margin: 0;
+    padding: 0;
+    background-color: #f4f4f4;
+}
+
+header {
+    background-color: #333;
+    color: #fff;
+    text-align: center;
+    padding: 1em;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+section {
+    margin: 20px;
+}
+
+h2 {
+    color: #333;
+}
+
+.pet-card {
+   
+    border: 1px solid #ddd;
+    border-radius: 8px;
+    overflow: hidden;
+    margin: 10px;
+    padding: 10px;
+    background-color: #fff;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+   
+}
+
+
+
+.pet-img {
+   
+    width: 100%;
+    height: 100vh;
+    object-fit: cover; 
+}
+header {
+    background-color: #333;
+    color: #fff;
+    text-align: center;
+    padding: 1em;
+}
+
+section#about {
+    margin: 20px;
+    padding: 20px;
+    background-color: #fff;
+    border-radius: 8px;
+    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+h2 {
+    color: #333;
+    border-bottom: 2px solid #333;
+    padding-bottom: 10px;
+    margin-bottom: 20px;
+}
+
+p {
+    line-height: 1.6;
+    margin-bottom: 15px;
+}
+
+ul {
+    margin-top: 10px;
+    margin-bottom: 20px;
+}
+
+li {
+    list-style-type: disc;
+    margin-left: 20px;
+}
+
+footer {
+    background-color: #333;
+   
+    text-align: center;
+    padding: 1em;
+    
+    bottom: 0;
+    width: 100%;
+}
+footer a{
+    
+    display: flex;
+    color: #9abd00;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+nav {
+    background-color: #555;
+    overflow: hidden;
+}
+
+nav a {
+    float: left;
+    display: block;
+    color: #fff;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+nav a:hover {
+    background-color: #ddd;
+    color: #333;
+}
+.logo{
+    width: 50px;
+    height: auto;
+    border-radius: 50%;
+}
+#sidebar {
+    background-color: #333;
+    color: #fff;
+    width: 100px;
+    opacity: 0.7;
+    height: 50vh;
+    position: fixed;
+    left: 0;
+    top: 25%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
```

Comment:
| Your Design|
|--------|
| ![image](https://github.com/sumairq/css-experiments/assets/60618877/2dd0f260-2fa0-47f0-98c1-8a54e93697cb)  |

- [ ] `postion: fixed;` is not appropriate in for this layout. Right now it is interfering with the content behind. It is much better to keep the sidebar within the flow of the document. Let's keep it at the left of text content.